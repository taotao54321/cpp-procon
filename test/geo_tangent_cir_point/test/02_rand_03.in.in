-125 55
22 -823 20
