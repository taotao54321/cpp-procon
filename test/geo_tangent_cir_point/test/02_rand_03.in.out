2.2053381624 -825.8585595560
41.6462693597 -819.2551234671
